src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java

```diff
@@ -643,8 +642,7 @@ public VoxelShape getCollisionShape()
 	@Override
 	public @NotNull Collection<ItemStack> getDrops(@NotNull ItemStack tool, Entity entity)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return drops;
```

Comment:
Probably a good idea to return a copy of the drops collection

Reply:
Copied.